packages/@o3r/rules-engine/project.json

```diff
@@ -27,6 +28,24 @@
         "tsConfig": "packages/@o3r/rules-engine/tsconfig.build.json"
       }
     },
+    "extract-rules-engine": {
+      "executor": "./packages/@o3r/rules-engine/dist:extractor",
+      "outputs": [
+        "{options.outputFactsDirectory}/rules.facts.metadata.json",
+        "{options.outputOperatorsDirectory}/rules.operators.metadata.json"
+      ],
+      "options": {
+        "tsConfig": "{projectRoot}/tsconfig.build.json",
```

Comment:
`{projectRoot}` should not work for this field, are you sure it is not defaulting to another `tsconfig.json`